docs/mkdocs.yml

```diff
@@ -70,23 +70,58 @@ nav:
   - Examples:
       - ./examples/index.md
       - Agents:
+          - ./examples/agent/.ipynb_checkpoints/anthropic_agent-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/custom_agent-bak-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/custom_agent-bak2-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/custom_agent-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/mistral_agent-bak-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/mistral_agent-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/multi_document_agents-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/multi_document_agents-v1-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/openai_agent-checkpoint.ipynb
+          - ./examples/agent/.ipynb_checkpoints/openai_agent_parallel_function_calling-checkpoint.ipynb
```

Comment:
probably we don't want to include these `.ipynb_checkpoints`?

Reply:
oof sorry